CI is failing with
```
10:46:20,252 INFO  - CMake Error at /buildagent/workspace/24733/zephyr/cmake/modules/zephyr_default.cmake:18 (cmake_minimum_required):
10:46:20,252 INFO  -   CMake 3.28.0 or higher is required.  You are running version 3.21.7
```
Probably because of this: https://github.com/zephyrproject-rtos/zephyr/pull/113911

Lukasz is on vacation till the end of this week. Adding @abonislawski.

--- a/west.yml
+++ b/west.yml
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 9885b002c1bdfd520355de44daa34edefc86f9e6
+      revision: fbd71c30f7b9083de9826577edb57d360efc01ed
       remote: zephyrproject
 
       # Import some projects listed in zephyr/west.yml@revision

